echo-rest/echo-rest.gradle

```diff
@@ -26,6 +26,8 @@ dependencies {
 
   implementation "io.spinnaker.kork:kork-web"
   implementation "com.fasterxml.jackson.core:jackson-databind"
+  implementation "org.hibernate.validator:hibernate-validator"
```

Comment:
For me, the way forward is to change
```
import org.hibernate.validator.constraints.NotEmpty;
```
to
```
import javax.validation.constraints.NotEmpty;
```
and then leave out this dependency.  The [javadoc](https://docs.jboss.org/hibernate/validator/6.0/api/index.html?org/hibernate/validator/constraints/NotEmpty.html) marks the hibernate annotation as deprecated.  It's not clear from that page, but "the standard NotEmpty" constraint is `javax.validation.constraints.NotEmpty`.

Reply:
Updated the code.